src/hooks/screen/useChatScreenHook.ts

```diff
@@ -17,7 +17,7 @@ import useLocalDatabaseHook from '../../database/hooks/useLocalDatabaseHook';
 import useUserAuthHook from '../core/auth/useUserAuthHook';
 import {CHANNEL_TYPE_GROUP, CHANNEL_TYPE_PERSONAL} from '../../utils/constants';
 import {getAnonymousUserId, getUserId} from '../../utils/users';
-import {randomString} from '../../utils/string/StringUtils';
+import {getOfficialAnonUsername, randomString} from '../../utils/string/StringUtils';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `getOfficialAnonUsername` is a good addition. It helps to centralize the logic for generating official anonymous usernames, which can improve maintainability and consistency across the codebase.
```diff
- import {randomString} from '../../utils/string/StringUtils';
+ import {getOfficialAnonUsername, randomString} from '../../utils/string/StringUtils';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->